src/components/Modal/index.tsx

```diff
@@ -175,6 +179,7 @@ class Modal extends React.Component<ThemedModalProps, any> {
 export interface ModalProps extends DefaultProps {
 	title?: string;
 	width?: ResponsiveStyle;
+	position?: 'center' | 'top';
```

Comment:
Let's add it to the README as well :)

Reply:
I thought we had automated docs, added.
Is there a place where theme properties are documented so I can add the header.height too?